deepchecks/checks/integrity/dominant_frequency_change.py

```diff
@@ -99,24 +99,24 @@ def _find_p_val(self, key: str, baseline_hist: Dict, test_hist: Dict, baseline_c
 
         return p_val
 
-    def _dominant_frequency_change(self, dataset: Dataset, baseline_dataset: Dataset,
+    def _dominant_frequency_change(self, train_dataset: Dataset, test_dataset: Dataset,
                                    feature_importances: pd.Series = None):
         """Run the check logic.
 
         Args:
-            dataset (Dataset): The dataset object. Must contain an index.
-            baseline_dataset (Dataset): The baseline dataset object. Must contain an index.
+            train_dataset (Dataset): The dataset object. Must contain an index.
+            test_dataset (Dataset): The baseline dataset object. Must contain an index.
         Returns:
             CheckResult: result value is dict that contains the dominant value change for each column.
         """
-        baseline_dataset = Dataset.validate_dataset_or_dataframe(baseline_dataset)
-        dataset = Dataset.validate_dataset_or_dataframe(dataset)
-        dataset.validate_shared_features(baseline_dataset)
+        test_dataset = Dataset.validate_dataset_or_dataframe(test_dataset)
+        train_dataset = Dataset.validate_dataset_or_dataframe(train_dataset)
+        train_dataset.validate_shared_features(test_dataset)
 
-        columns = baseline_dataset.features
+        columns = test_dataset.features
 
-        test_df = dataset.data
-        baseline_df = baseline_dataset.data
+        test_df = train_dataset.data
+        baseline_df = test_dataset.data
```

Comment:
Here is the switch test=test and baseline=train

Reply:
updated